storage-proofs/src/circuit/multi_proof.rs

```diff
@@ -42,4 +42,10 @@ impl<'a, E: Engine> MultiProof<'a, E> {
         }
         Ok(())
     }
+
+    pub fn to_vec(&self) -> Vec<u8> {
+        let mut out = Vec::new();
+        self.write(&mut out).expect("known allocation target");
```

Comment:
QUESTIONS:

1. Are you sure you want to panic here?
2. Instead of adding this method, could you instead satisfy the `From` trait?

Reply:
> Why do you defile the holy `MultiProof` so?

@porcuquine Is this a real questions?

Reply:
> Are you sure you want to panic here?

This can't panic, this is how write works, the write trait only panics if the underlying storage is full, which does not happen for a `Vec`. 

> Instead of adding this method, could you instead satisfy the From trait?

The way the std lib and most libraries do conversions to `Vec` is using a `to_vec` method, `Into` and `From` are for consuming the thing, where as `to_vec` only takes a reference to `self`.

Reply:
@dignifiedquire It was a real question, which I found the answer to myself eventually. The form of the question was just to amuse you as you fight through getting this finished.